<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModesWidget</class>
 <widget class="QWidget" name="ModesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>245</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>245</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Режимы</string>
  </property>
  <widget class="QPushButton" name="m_generAEP_pb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Генератор АЭП</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_generPEMI_pb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Генератор ПЭМИН</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_timerAEP_pb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Таймер АЭП</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_search_pb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Поисковичок</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_timerPEMI_pb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Таймер ПЭМИ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
